enhancements/security/scc-csi-volumes.md

```diff
@@ -0,0 +1,234 @@
+---
+title: scc-csi-volumes
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@bparees"
+  - "@jsafrane"
+approvers:
+  - "@sttts"
+  - "@deads2k"
+creation-date: 2021-04-27
+last-updated: 2021-04-27
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Security Context Constraints for CSI Volumes
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Ephemeral CSI volumes are a newer type of volume in Kubernetes, which allows content to mounted into pods with underlying ephemeral file systems.
```

Comment:
Are "ephemeral" CSI volumes distinct from "CSI volumes" in general, particularly as regards the SCC changes being proposed here?   My impression is no, but this EP specifically states "ephemeral" a lot, implying to me some sort of difference.